packages/rancher-sachet/charts/templates/service.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Service
+metadata: {{- include "sachet.metadata" . | nindent 2 }}
+spec:
+  type: {{ .Values.service.type }}
```

Comment:
service of type NodePort needs to be able to specify nodePort in YAML below

https://kubernetes.io/docs/concepts/services-networking/service/#nodeport

Reply:
The support for nodePort type is added. 
With this change, users need to make sure the node port they provide is a valid one.